tests/test_resnet.py

```diff
@@ -0,0 +1,128 @@
+from case_script import TestPythonScript
+import os
+import numpy as np
+import cv2
+import shutil
+import tarfile
+import errno
+from six.moves import urllib
+import sys
+
+
+CAFFE_ILSVRC12_URL = "http://dl.caffe.berkeleyvision.org/caffe_ilsvrc12.tar.gz"
+
+
+# this is duplicated code, but this file should be self-contained
```

Comment:
Sure. I currently thinking about using FakeData by a flag in the ResNet example, which simplifies this testing.